### ISSUE TITLE
Hi, the performance of PPI cannot match the paper

### ISSUE THREAD

**Issue by Dreamer312** [RL4M/KeAP#2]
Hi, most experiments in the paper can be reproduced by your pretrained model. However the performance of PPI is lower than paper. I just set the path as the readme mentioned. Could you give me some intuition？

SHS27K BFS :  epoch: 299, Training_avg: label_loss: 0.03999417771895727, recall: 0.9771298891773549, precision: 0.98057270795867, F1: 0.9788472239560381, Validation_avg: loss: 0.5734837651252747, recall: 0.6479085164008231, precision: 0.6976668826960241, F1: 0.6718676860165164, Best valid_f1: 0.6973825194662955, in 285 epoch

SHS27K DFS : epoch: 299, Training_avg: label_loss: 0.04056152577201525, recall: 0.9721116754062944, precision: 0.9806885877418033, F1: 0.9763762664752628, Validation_avg: loss: 1.2854664325714111, recall: 0.6594566077551923, precision: 0.6490134994807724, F1: 0.6541933795129848, Best valid_f1: 0.6974584554732253, in 179 epoch

SHS148K BFS : epoch: 299, Training_avg: label_loss: 0.0827140137553215, recall: 0.9410505292185909, precision: 0.9569001914733489, F1: 0.9489012241137775, Validation_avg: loss: 0.6191256761550903, recall: 0.4972978053479934, precision: 0.7124007001481034, F1: 0.5857249605139444, Best valid_f1: 0.6378486157125448, in 62 epoch

SHS148K DFS : epoch: 299, Training_avg: label_loss: 0.08096629463964039, recall: 0.9430068889223355, precision: 0.9583513056764192, F1: 0.9506124661548588, Validation_avg: loss: 0.4336592793464661, recall: 0.723633782824695, precision: 0.8059229593718671, F1: 0.762564818458318, Best valid_f1: 0.8203462674404631, in 140 epoch

STRING BFS  :epoch: 299, Training_avg: label_loss: 0.09552797081013178, recall: 0.9200262255716597, precision: 0.9468857032485015, F1: 0.9332547471275457, Validation_avg: loss: 0.539549797259528, recall: 0.5116038484085657, precision: 0.7283021337871446, F1: 0.6010168186458669, Best valid_f1: 0.6643095854578324, in 16 epoch

**Comment by yunxiangfu2001** [comment]
Hi,

I'm not sure what's wrong.

Could you please re-run an SHS27K or SHS148 experiment using the train-test splits for SHS27k and SHS148k from https://drive.google.com/drive/folders/1oPmnNQGP18g7zc2lbnuMNBADSRxyG_ne?usp=sharing ? You need to adjust the paths of training (https://github.com/RL4M/KeAP/blob/main/src/benchmark/GNN_PPI/run.py#L33) and testing (https://github.com/RL4M/KeAP/blob/main/src/benchmark/GNN_PPI/run_test.py#L24).

**Comment by Dreamer312** [comment]
Sure, I will try it, thanks!
